aws/resource_aws_api_gateway_method.go

```diff
@@ -226,6 +236,22 @@ func resourceAwsApiGatewayMethodUpdate(d *schema.ResourceData, meta interface{})
 		})
 	}
 
+	if d.HasChange("request_validator_id") {
+		var request_validator_id *string
+		if v, ok := d.GetOk("request_validator_id"); ok {
+			// requestValidatorId cannot be an empty string; it must either be nil
+			// or it must have some value. Otherwise, updating fails.
+			if s := v.(string); len(s) > 0 {
+				request_validator_id = &s
+			}
+		}
```

Comment:
I checked this out and tried manually, it seems we're sending `nil` in the operations below which is effectively doing what we want and removing it, so I guess this is find 👍